# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/DeleteTextTransaction.cpp
# Commit: b9ebe58001d7
# Full Hash: b9ebe58001d787bb0bbad6d39ae8681966d5a77f
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2020-04-04 09:29:13
# Regressor Bug: 1619914
# File Overlap Count: 1
# Description:
#   Bug 1619914 - part 2: Mark transaction class methods and their caller methods as `MOZ_CAN_RUN_SCRIPT r=m_kato
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D69303
# ==============================================================================

diff -r ce2087ddab5c -r b9ebe58001d7 editor/libeditor/DeleteTextTransaction.cpp
--- a/editor/libeditor/DeleteTextTransaction.cpp	Fri Apr 03 08:30:37 2020 +0000
+++ b/editor/libeditor/DeleteTextTransaction.cpp	Fri Apr 03 08:32:01 2020 +0000
@@ -97,8 +97,7 @@
   return mEditorBase->IsModifiableNode(*mTextNode);
 }
 
-MOZ_CAN_RUN_SCRIPT_BOUNDARY NS_IMETHODIMP
-DeleteTextTransaction::DoTransaction() {
+NS_IMETHODIMP DeleteTextTransaction::DoTransaction() {
   if (NS_WARN_IF(!CanDoIt())) {
     return NS_ERROR_NOT_AVAILABLE;
   }
@@ -140,8 +139,7 @@
 
 // XXX: We may want to store the selection state and restore it properly.  Was
 //     it an insertion point or an extended selection?
-MOZ_CAN_RUN_SCRIPT_BOUNDARY NS_IMETHODIMP
-DeleteTextTransaction::UndoTransaction() {
+NS_IMETHODIMP DeleteTextTransaction::UndoTransaction() {
   if (NS_WARN_IF(!CanDoIt())) {
     return NS_ERROR_NOT_AVAILABLE;
   }